### DISCUSSION TITLE
Please add PNPM support.

### DISCUSSION THREAD

**Discussion by MohdAhmad1** [vercel/next.js#26672] [Ideas]
### Describe the feature you'd like to request

When creating a NEXT JS app, the ` create-next-app ` gives two package-manages to choose from, i.e `YARN & NPM`.
As we know that PNPM saves disk space by caching npm packages in a local store and then linking them to your package.json.
This increases its speed and saves disk space, as we Indians do not have much internet facility, this will be very helpful for us

### Describe the solution you'd like

The solution would be like

```shell
 [npx | pnpx] create-next-app --ts -e [name]|[github-url] [ --use-npm | --use-pnpm ] 
```

OR
```shell
 yarn create next-app --ts -e [name] | [github-url] [ --use-npm | --use-pnpm ]
```

### Describe alternatives you've considered

I have not considered any alternatives of my solution, but pnpm stand a chance against npm and yarn

**Comment by admmasters** [comment]
I’m of the opinion that pnpm is a superior package manager to npm/yarn and due to its relative strictness it finds subtle bugs in package.json deps/devDeps, but like you say it’s faster than both npm and yarn as well. Yarn 1 is basically in maintenance mode only now.

**Reply by MohdAhmad1** [reply]
I am using next js with pnpm and it is working fine

**Comment by kotx** [comment]
It would be good to get an option to not install packages as well, so the user can use their package manager of choice on their own.

**Reply by MohdAhmad1** [reply]
Or, support of remote cdn packages such as [skypack](https://www.skypack.dev/)
